### DISCUSSION TITLE
Questions on VTK and VTU Export Fields

### DISCUSSION THREAD

**Discussion by mikerife** [ansys/pydpf-core#1299] [Q&A]
Hi 
In the Operator help for both [VTK](https://dpf.docs.pyansys.com/version/stable/operator_reference_load.html) and VTU Export the input pins for the fields is given as "2, 3..." which to me implies that more than two fields are allowed to be written.  But it is pretty clear from the code and testing out the operator that the limit is two fields.  Is this a documentation mistake or is the plan to support writing more than two fields?  If the plan is to only support writing two fields, why two?  Why not one?  Or five?

BTW the PyDPF Core API Reference [doc](https://dpf.docs.pyansys.com/version/stable/api/ansys.dpf.core.operators.serialization.vtk_export.html#) clearly shows support for just two fields.

Second question is that to me the ability to define a name for the field would be useful (the name in the vtk object).  Does this seem like a good idea to anyone else?  I know that we can use PyVista functionality to rename the field name.  But would be nice to be able to change it prior to vtk file creation.

Mike

**Comment by susannehaase** [comment]
Hi, is there any update regarding this topic? Susanne

**Comment by felixschurk** [comment]
It may not be a proper answer for this question, but there is [migrate_to_vtu](https://dpf.docs.pyansys.com/version/stable/api/ansys/dpf/core/operators/serialization/migrate_to_vtu/index.html) which takes every field available and writes vtu files. Similar there is also a [migrate_file_to_vtk](https://dpf.docs.pyansys.com/version/stable/api/ansys/dpf/core/operators/serialization/migrate_file_to_vtk/index.html) function.
